src/js/components/Header/stories/Responsive.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Header, Anchor, Box, ResponsiveContext, Menu } from 'grommet';
+import { Grommet as GrommetIcon, Menu as MenuIcon } from 'grommet-icons';
+import { grommet } from 'grommet/themes';
+
+const Responsive = () => (
+  <Grommet theme={grommet}>
+    <Header background="light-4" height="xsmall">
+      <Anchor
+        margin={{ left: 'medium', vertical: 'xsmall' }}
+        href="https://tools.grommet.io/"
+        icon={<GrommetIcon color="brand" />}
+        label="Grommet Tools"
+      />
+      <ResponsiveContext.Consumer>
+        {size =>
+          size === 'small' ? (
+            <Box alignSelf="end" margin="large">
```

Comment:
Not sure why this margin is needed. we can remove it and simplify it, same for line #44.